server/blob/blob-deduplicating/src/main/scala/org/apache/james/server/blob/deduplication/GCJsonReporter.scala

```diff
@@ -44,14 +44,14 @@ object JsonReport {
                    `garbage-collection-iterations`: TreeSet[String],
                    blobs: Seq[BlobId],
                    references: Seq[Reference],
-                   deletions: Seq[Dereference])
+                   dereferences: Seq[Dereference])
 
 
   //action
   implicit val relatedActionWrites: Writes[RelatedAction] = {
     case Init => JsString("init")
     case Save(blobId, reference) => JsString(s"save(blob = ${blobId.asString()}, reference = ${reference.id})")
-    case Delete(reference) => JsString(s"delete(reference = ${reference.id})")
+    case DereferenceAction(reference) => JsString(s"dereference(reference = ${reference.id})")
```

Comment:
Why add an Action suffix here and not above?

Reply:
It was to disembiguate between the different `Dereference` class but i think i will just prefix with the RelatedAction object.